restlight-test/restlight-test-starter/src/main/java/io/esastack/restlight/test/autoconfig/MockMvcAutoConfiguration.java

```diff
@@ -36,4 +39,14 @@
     public MockMvc mockMvc(ApplicationContext context) {
         return MockMvcBuilders.contextSetup(context);
     }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public RestlightOptions options() {
```

Comment:
Maybe the `AutoMockMvcOptions` should be autowired by `application.properties`  which is as the same way as
`AutoRestlightServerOptions`?